app/services/bill-runs/two-part-tariff/review-bill-run.service.js

```diff
@@ -20,14 +20,15 @@ const ReviewBillRunPresenter = require('../../../presenters/bill-runs/two-part-t
  * details of the bill run and the licences linked to it as well as any data that has been used to filter the results.
  */
 async function go (id, page, yar) {
-  const { filterIssues, filterLicenceHolderNumber, filterLicenceStatus } = _getFilters(id, yar)
+  const { filterIssues, filterLicenceHolderNumber, filterProgress, filterLicenceStatus } = _getFilters(id, yar)
```

Comment:
With our standard of putting things in alphabetical order, I think we need to re-arrange this
```suggestion
  const { filterIssues, filterLicenceHolderNumber, filterLicenceStatus, filterProgress } = _getFilters(id, yar)
```